community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/resume.py

```diff
@@ -494,7 +495,7 @@ def create_placements(nodes: List[str], excl_job_id:Optional[int], lkp: util.Loo
     return placements
 
 
-def _allocate_nodes_to_placements(nodes: List[str], excl_job_id:Optional[int], lkp: util.Lookup) -> List[PlacementAndNodes]:
+def _allocate_nodes_to_placements(nodes: List[str], max_distance, excl_job_id:Optional[int], lkp: util.Lookup) -> List[PlacementAndNodes]:
```

Comment:
We should take `max_distance` from `nodeset`, not take it as an argument.

Reply:
Done